EventsExpress/ClientApp/src/components/event/event-item-view.js

```diff
@@ -272,6 +290,53 @@ export default class EventItemView extends Component {
                                 <div className="text-box overflow-auto shadow p-3 mb-5 mt-2 bg-white rounded">
                                     {description}
                                 </div>
+                                <div className="text-box overflow-auto shadow p-3 mb-5 mt-2 bg-white rounded">
```

Comment:
Please move this inventory display to separate component.

Reply:
Solved